.env

```diff
@@ -1,8 +0,0 @@
-FACEME_API_KEY=zycmoKBnCECUfRiY55-sI1uKlPtCcET
-FACEME_API_SECRET=Ics6CXPwIWh9EP4hynkt-9-izcgWS65K
-FACE_SET_TOKEN=81b5124d27a3b5c138c9624894c4bc73
-REACT_APP_AWS_ACCESS_KEY_ID=AKIAJUYIVNVFQU66WDAA
-AWS_ACCESS_KEY_ID=AKIAJUYIVNVFQU66WDAA
-AWS_SECRET_ACCESS_KEY=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
```

Comment:
Might want to regenerate all these tokens for security/$$$$.

Reply:
haha yes. this was definitely my bad.